### DISCUSSION TITLE
A1111 Tagcomplete Broken

### DISCUSSION THREAD

**Discussion by gitupp22** [DominikDoom/a1111-sd-webui-tagcomplete#303] [Q&A]
Today when typing in A1111, I get strange symbols instead of the words usually given. When typing the number 1, instead of something like 1girl popping up, it says �?���U��qk�1s5v��vV��1��ccJ4�ذ�<������d\[�}��f��ݶ����D�V'�, 

The only tags that seem to work are the quality tags from extra-quality-tags.csv. It's a small list, so I decided to try and delete all tags except a few from the danbooru.csv file and it works as long as there are only about 5 tags in there.

I tried reinstalling. It is still working for me in ForgeUI, but not in Automatic1111.
![image](https://github.com/user-attachments/assets/0961a662-1e11-4305-a3bb-04841e1b9e4a)

**Comment by DominikDoom** [comment]
I created an issue for this discussion, that makes it easier for me to track. I'll continue there: https://github.com/DominikDoom/a1111-sd-webui-tagcomplete/issues/304
